### DISCUSSION TITLE
Photos taken at home are titled with business at same location

### DISCUSSION THREAD

**Discussion by PaulRitzkat** [photoprism/photoprism#1396] [Q&A] [answered]
Hi, nearly all photos taken at home are named after the physiotherapy business at the same address and have corresponding keywords. Only some of the many cat pictures are labeled "cat" and everything else, from cocktails to barbecue, is named and labeled after the business.

Is there a way to set a home location, or change labes based on the geo tag?

Thanks!

**Comment by lastzero** [comment, marked as answer]
It's an edge case. Note that we only resolve locations in two dimensions. We may later implement a UI to perform changes. It's technically possible, just not implemented yet.

**Reply by PaulRitzkat** [reply]
Thanks! Naming or creating geotags would be a great feature.
